instrumentation/sequel/lib/sequel/extensions/opentelemetry.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'sequel'
+
+# Sequel: The Database Toolkit for Ruby
+module Sequel
+  # The Sequel::Opentelemetry extension
+  #
+  # @example Simple usage
+  #
+  #     require 'sequel-opentelemetry'
+  #     DB.extension :opentelemetry
+  #
+  # Related module: Sequel::Opentelemetry
+  module Opentelemetry
+    def tracer
+      @tracer ||= ::OpenTelemetry.tracer_provider.tracer('Sequel', ::Sequel::VERSION)
+    end
+
+    def trace_execute(super_method, sequel_method, sql, options, &block)
+      response = nil
+      attributes = {
+        'component' => 'Sequel',
+        ::OpenTelemetry::SemanticConventions::Trace::DB_NAME => opts[:database],
+        ::OpenTelemetry::SemanticConventions::Trace::DB_SYSTEM => database_type.to_s,
+        ::OpenTelemetry::SemanticConventions::Trace::DB_STATEMENT => sql,
+        ::OpenTelemetry::SemanticConventions::Trace::DB_OPERATION => sequel_method
```

Comment:
This must correspond to the type of SQL statement being performed, i.e. `SELECT`, `INSERT`, etc...

https://opentelemetry.io/docs/specs/semconv/database/database-spans/#call-level-attributes

Reply:
Removed this as well since it's not available for all Sequel adapters